src/python/pants/bin/pants_runner.py

```diff
@@ -34,7 +34,9 @@ def __init__(self, exiter, args=None, env=None, start_time=None):
   def run(self):
     options_bootstrapper = OptionsBootstrapper(env=self._env, args=self._args)
     bootstrap_options = options_bootstrapper.get_bootstrap_options()
-    ExceptionSink.set_destination(bootstrap_options.for_global_scope().pants_workdir)
+    ExceptionSink.reset_log_location(LogLocation.from_options_for_current_process(
```

Comment:
as we discussed, separating out pid and log location between daemon_runner and pants_runner is

Reply:
This has been changed to just update the location and the exiter from the bootstrap options.